src/encoded/tests/test_indexing.py

```diff
@@ -325,3 +326,23 @@ def test_load_and_index_perf_data(testapp, indexer_testapp):
                                                                             frame_time, embed_time))
     # userful for seeing debug messages
     # assert False
+
+
+def test_permissions_database_applies_permissions(award, lab, file_formats, wrangler_testapp, anontestapp, indexer_testapp):
+    """ Tests that anontestapp gets view denied when using datastore=database """
+    from time import sleep
+    file_item_body = {
+        'award': award['uuid'],
+        'lab': lab['uuid'],
+        'file_format': file_formats.get('fastq').get('uuid'),
+        'paired_end': '1',
+        'status': 'released'
+    }
+    res = wrangler_testapp.post_json('/file_fastq', file_item_body, status=201).json
+    item_id = res['@graph'][0]['@id']
+    indexer_testapp.post_json('/index', {'record': True})
+    sleep(1)  # let es catch up
+    res = anontestapp.get('/' + item_id).json
+    assert res['file_format'] == {'error': 'no view permissions'}
+    res = anontestapp.get('/' + item_id + '?datastore=database').json
```

Comment:
Do these endpoint calls with `.json` on their result not need `format=json` in the query param part?

Reply:
`get` will return a `Response` object who has a `json` member that is the request in JSON form. I do not believe there should be any difference if `format=json` is added or not (just tried it on local 😄 ).

Reply:
OK